wallets/core/src/namespaces/evm/builders.ts

```diff
@@ -17,3 +23,38 @@ export const canEagerConnect = () =>
   new ActionBuilder<EvmActions, 'canEagerConnect'>('canEagerConnect');
 export const canSwitchNetwork = () =>
   new ActionBuilder<EvmActions, 'canSwitchNetwork'>('canSwitchNetwork');
+
+// Hooks
+export const changeAccountSubscriber = (getInstance: () => ProviderAPI) =>
+  new ChangeAccountSubscriberBuilder<string[], void, ProviderAPI, EvmActions>()
```

Comment:
```suggestion
  new ChangeAccountSubscriberBuilder<string[], undefined, ProviderAPI, EvmActions>()
```
Shouldn't we do this?